integrative/chromHMM/jobSubmission/bootstrapBinarise.sh

```diff
@@ -46,7 +46,7 @@ then
 		#change project into folder name for binarised folder 
 		DIR=$(echo $PROJECT | tr / _)
 
-		cd $SLURM_SUBMIT_DIR
+        cd $SLURM_SUBMIT_DIR
```

Comment:
I am assuming this is needed for the subsequent source command

Reply:
probably